main.py

```diff
@@ -19,20 +19,20 @@
 return: string representing output code or error message
 """
 def main(input_code, input_lang, output_lang):
-    # TODO: eventually replace this with 400 error
+
     if input_lang == None and output_lang == None:
         abort(400, "Error: must specify input and output languages")
-        input_lang = "js"
-        output_lang = "py"
 
-    #TODO: also do error checking on the output language
     if input_lang == "js":
         gast = js_main.js_to_gast(input_code)
     elif input_lang == "py":
         gast = py_main.py_to_gast(input_code)
     else:
         #TODO: figure out hwo to do error messages
-        return "Error must specify language. For example, js for javascript and py for python"
+        return "Error must specify input language. For example, js for javascript and py for python"
+
+    if output_lang != "js" and output_lang != "py":
```

Comment:
I feel like it would be easier to have an array of supported language symbols `["js", "py"]` and doing an `includes` check for the `input_lang` and `output_lang` at the start of the method.